app/views/training_and_support/show.html.erb

```diff
@@ -0,0 +1,9 @@
+<% if @provider.name == "Ambition Institute" %>
+  <h1 class="govuk-heading-l">Guidance for Ambition Institute</h1>
+<% elsif @provider.name == "Education Development Trust" %>
+  <h1 class="govuk-heading-l">Guidance for Education Development Trust</h1>
+<% elsif @provider.name == "Teach First" %>
+  <h1 class="govuk-heading-l">Guidance for Teach First</h1>
+<% elsif @provider.name == "UCL" %>
+  <h1 class="govuk-heading-l">Guidance for UCL</h1>
+<% end %>
```

Comment:
I know we don't have much in them, buuuuut...

Could you make a partial for each of them? And in this template, just render the correct partial, no contents? I think that will have to happen as soon as we start adding content anyway.